recipes/libprotobuf/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libprotobuf" %}
+{% set version = "3.3.0" %}
+{% set sha256 = "94c414775f275d876e5e0e4a276527d155ab2d0da45eed6b7734301c330be36e" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/google/protobuf/archive/v{{ version }}/protobuf-v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9                # [win and py27]
+    - vc10               # [win and py34]
+    - vc14               # [win and py35]
+    - vc14               # [win and py36]
+
+requirements:
+  build:
+    - git
+    - cmake              # [win]
+    - autoconf           # [unix]
+    - automake           # [unix]
+    - libtool            # [unix]
+    - pkg-config         # [unix]
+    - zlib 1.2.*
+    - python             # [win]
+  run:
+    - zlib 1.2.*
+
+test:
+  commands:
+    - protoc --help
+    - test -f ${PREFIX}/lib/libprotobuf.a                           # [unix]
+    - test -f ${PREFIX}/lib/libprotobuf.dylib                       # [osx]
+    - test -f ${PREFIX}/lib/libprotobuf.so                          # [linux]
+    - if not exist %PREFIX%\\Library\\lib\\libprotobuf.lib exit 1   # [win]
+
+about:
+  home: https://developers.google.com/protocol-buffers/
+  license: BSD 3-Clause
```

Comment:
I know we didn't have it before, but we should add `license_file: LICENSE` to package the license file from the source code.